decidim-assemblies/app/commands/decidim/assemblies/admin/create_assembly_admin.rb

```diff
@@ -27,7 +27,7 @@ def call
           return broadcast(:invalid) if form.invalid?
 
           ActiveRecord::Base.transaction do
-            create_or_invite_user
+            @user = @user ||= existing_user || new_user
```

Comment:
Unintended?

Reply:
LOL?

Reply:
Lol, definitely unintended. Seems to work though xDDD. Will fix in a further PR

Reply:
@deivid-rodriguez we have inconsistencies between `tslint` and `eslint`. I plan to move all this to `prettier` (that supports both file formats) in a further PR but unfortunately I don't have time right now to deal with it. I'll try to do it soon, though!

Reply:
Although they are different syntaxes, I guess it makes sense to normalize common rules :+1:. But please do it in a separate PR so we can discuss it instead of sneaking it in a ~ +13K, -7K one! :laughing:

Reply:
> Lol, definitely unintended. Seems to work though xDDD. Will fix in a further PR

I've actually seen a similar technique somewhere to avoid redefinition warnings, so I wasn't sure it wasn't some trick of yours :smile:

Reply:
> I plan to move all this to prettier (that supports both file formats) in a further PR but unfortunately I don't have time right now to deal with it. I'll try to do it soon, though!

Sorry for triple posting, but... could you actually open a ticket first for discussion? I haven't hit any walls with `eslint` + `tslint`, and moving to `prettier` would imply reconfiguring my editor to play nice with it or stop having JS syntax checking until I do so, so I'd like to hear some motivations before the change!